src/main/java/com/google/sps/data/GovPostReq.java

```diff
@@ -18,8 +20,21 @@
  * The adaptors get their information from their respective data sources - the government API is called here. 
  */
 public class GovPostReq {
-  
-  public  ArrayList<NSWGovAQDataPoint> extractAQI() throws Exception {
+  public static void main(String[] args) {
```

Comment:
Why is there a `main` here? Testing. This main doesn't do anything with the class, why is it here at all?

Reply:
this file is deleted in the latest push